### PR TITLE
Simple first dummy UI

### PR DESCRIPTION
**Pull request by da-kami**
image

### CODE REVIEW

File: frontend/src/App.test.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import App from "./App";
+
+test("renders learn react link", () => {
+    render(<App />);
+    const linkElement = screen.getByText(/learn react/i);
+    expect(linkElement).toBeInTheDocument();
+});
```
Comment (klochowicz) [inline frontend/src/App.test.tsx:9]: Perhaps we could remove the stale test as it wouldn't pass anyway?

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                                            <Box>Entry</Box>
+                                            <Box>Entry</Box>
+                                        </Box>
+                                    </VStack>
+                                </Flex>
+                                <Flex width={"50%"} marginLeft={5}>
+                                    <VStack spacing={5} shadow={"md"} padding={5} align={"stretch"}>
+                                        <HStack>
+                                            <Text align={"left"}>Current Price:</Text>
+                                            <Text>490000</Text>
```
Comment (klochowicz) [inline frontend/src/App.tsx]: nit: that's a very optimistic price :D

**Review by klochowicz — APPROVED** [review]
looks like a good skeleton for the UI!

**Comment by da-kami** [comment]
closing this for now, we will open a separate PR for frontend + daemon shortly

**Pull request closed without merging**